torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -194,6 +195,65 @@ def encode_forward(self, feat: Tensor) -> Tensor:
         return x
 
 
+class MultiCategoricalEmbeddingEncoder(StypeEncoder):
+    r"""An embedding look-up based encoder for multi_categorical features. It
+    applies :class:`torch.nn.EmbeddingBag` for each categorical feature and
+    concatenates the output embeddings.
+    Args:
+        out_channels (int, optional): Size of each embedding vector.
+            (default: :obj:`None`)
+        mode (str): "sum", "mean" or "max".
+            Specifies the way to reduce the bag. (default: :obj:`mean`)
+    """
+    supported_stypes = {stype.multicategorical}
+
+    def __init__(
+        self,
+        out_channels: Optional[int] = None,
+        stats_list: Optional[List[Dict[StatType, Any]]] = None,
+        stype: Optional[stype] = None,
+        post_module: Optional[Module] = None,
+        na_strategy: Optional[NAStrategy] = None,
```

Comment:
I assume we would need to error out in case `na_strategy` is set to something else than `NAStrategy.ZERO`.

Reply:
Good point.

Reply:
Acutally, `NAStrategy.ZERO` is the same as `NAStrategy.MOST_FREQUENT` because the most frequent class will always be mapped to 0.

Reply:
This is not true, right? `ZERO` would produce a zero embedding, while `MOST_FREQUENT` should pick up the embedding from the most frequent category.